frontend/src/metabase/entities/containers/EntityListLoader.jsx

```diff
@@ -1,5 +1,5 @@
-/* eslint-disable react/prop-types */
 import React from "react";
+import PropTypes from "prop-types";
```

Comment:
There have been teammates in favor of doing the propTypes in a separate PR.

Justification is that any regressions found with `git bisect` would point to a more focused PR.

I see that.

At the same time, at least as far as I can tell right now the risk of regressions due to adding propTypes is low.

So my vote is in favor.